src/equations/compressible_moist_euler_2d_lucas.jl

```diff
@@ -38,9 +37,8 @@ function CompressibleMoistEulerEquations2D(; g = 9.81, RealT = Float64)
     gamma = c_pd / c_vd # = 1/(1 - kappa)
     kappa = 1 - inv(gamma)
     L_00 = 3147620.0
```

Comment:
```suggestion
    L_00 = 3147620
```
There are a few other places missing - I only corrected them where I saw them, but surely missed a few